src/components/Card.svelte

```diff
@@ -24,16 +24,17 @@
 
 		return artistsOneline;
 	}
-	//when collapsed, pause audio and set expand indicator to false
 	function toggleExpand():void{
 		if(expand){
 			audio.pause();
 		}
+		savedPosition = trackPosition
+		console.log(trackPosition)
 		expand = !expand
 	}
 </script>
-
-<div class="card" on:click={toggleExpand} on:keyup={toggleExpand} aria-expanded={expand}>
+<!-- todo: test below with tapping on touchscreen -->
+<div class="card" on:click={toggleExpand} on:keypress={toggleExpand} aria-expanded={expand}>
```

Comment:
keypress is deprecated; see [mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event)